toml.md

```diff
@@ -259,7 +259,8 @@ String
 ------
 
 There are four ways to express strings: basic, multi-line basic, literal, and
-multi-line literal. All strings must contain only valid UTF-8 characters.
+multi-line literal. All strings must contain only valid Unicode characters.
+Strings should be implemented as UTF-8 byte sequences.
```

Comment:
```suggestion
multi-line literal. All strings must contain only valid UTF-8 encoded
Unicode characters.
```

Reply:
The change from "Unicode characters" to "UTF-8 encoded Unicode characters" is unnecessary because the top of the spec says "A TOML file must be a valid UTF-8 encoded Unicode document."

Can you explain why you removed "Strings should be implemented as UTF-8 byte sequences."? As noted in my original comment, other parts of the spec say how output should be implemented.

Reply:
Fair. I've editted to only remove the second suggestion then! :)

See https://github.com/toml-lang/toml/issues/890 for why I'm opposed to using "byte sequences" as the language here.